account-kit/smart-contracts/src/ma-v2/client/client.test.ts

```diff
@@ -2,3 +1,12 @@
+import {
+  custom,
+  parseEther,
+  publicActions,
+  getContract,
+  keccak256,
+  toHex,
```

Comment:
⚠️ **[eslint]** <[@typescript-eslint/no-unused-vars](https://typescript-eslint.io/rules/no-unused-vars)> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'toHex' is defined but never used.
<!-- __reviewdog__:ChBiNWQxOWM3Yzk1Y2UxYTE0EgZyZGpzb24= -->